wlauto/workloads/geekbench/__init__.py

```diff
@@ -87,6 +87,14 @@ class Geekbench(AndroidUiAutoBenchmark):
         Parameter('times', kind=int, default=1,
                   description=('Specfies the number of times the benchmark will be run in a "tight '
                                'loop", i.e. without performaing setup/teardown inbetween.')),
+        Parameter('iteration_timeout', kind=int, default=900,
+                  description='Timeout for a single iteration of the benchmark. This value is '
+                              ' multiplied by ''times'' to calculate the overall run timeout. '),
```

Comment:
Don't need leading space on this line since there is a trailing space at the end of the previous line.